contracts/STokenManager.sol

```diff
@@ -0,0 +1,96 @@
+// SPDX-License-Identifier: MPL-2.0
+pragma solidity 0.8.4;
+
+import {ERC721} from "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import {Counters} from "@openzeppelin/contracts/utils/Counters.sol";
+import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
+import {ISTokensManager} from "@devprotocol/i-s-tokens/contracts/interface/ISTokensManager.sol";
+import {IAddressConfig} from "./IAddressConfig.sol";
+import {STokensManagerStorage} from "./STokensManagerStorage.sol";
+import {ISTokensDescriptor} from "./ISTokensDescriptor.sol";
+
+contract STokensManager is
+	ISTokensManager,
+	STokensManagerStorage,
+	ERC721,
+	Ownable
+{
+	using Counters for Counters.Counter;
+	Counters.Counter private _tokenIds;
+	address public config;
+	address public descriptor;
+
+	// TODO この名前でいいか確認する
+	constructor(address _config) ERC721("SDev", "SDEV") {
+		config = _config;
+	}
+
+	modifier onlyLockup() {
+		require(
+			IAddressConfig(config).lockup() == _msgSender(),
+			"illegal access"
+		);
+		_;
+	}
+
+	function setDescriptor(address _descriptor) external onlyOwner {
+		descriptor = _descriptor;
+	}
```

Comment:
Remove

Reply:
https://github.com/dev-protocol/s-tokens/pull/1#discussion_r693687910

Reply:
https://github.com/dev-protocol/s-tokens/pull/1#discussion_r693687910

For the same reason.